src/rc/start-stop-daemon.c

```diff
@@ -59,6 +59,11 @@ static struct pam_conv conv = { NULL, NULL};
 #include <sys/capability.h>
 #endif
 
+#ifdef __linux__
+#include <linux/sched.h>
```

Comment:
seems this is only included for `SCHED_BATCH` ?  but glibc provides this for us, albeit only when `_GNU_SOURCE` is defined.  we try to avoid defining `_GNU_SOURCE`, but we should linux/ includes more.

maybe at the top do:

```
#ifdef __linux__
/* For extra SCHED_* defines. */
# define _GNU_SOURCE
#endif
```